mongodb/ipnskeys.go

```diff
@@ -13,6 +13,7 @@ import (
 type IPNSKey struct {
 	Name      string
 	Cid       string
+	Path      string
 	ThreadID  thread.ID
 	CreatedAt time.Time
 }
```

Comment:
wondering if it would be handy to grab UpdatedAt? could be cool someday to hold a path history to show in the dash, though probably lives on the bucket itself.